stok.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const Hapi = require('hapi');
+
+class Stok {
+  constructor(options) {
+    this.options = options;
+  }
+
+  createServer(routes) {
+    return new Promise((resolve, reject) => {
+      const server = new Hapi.Server();
+      server.connection({ port: this.options.web.port });
```

Comment:
`Stok` is just the web server, right? So the options are already limited to the web context.